packages/template-retail-react-app/config/default.js

```diff
@@ -22,8 +22,8 @@ module.exports = {
         commerceAPI: {
             proxyPath: `/mobify/proxy/api`,
             parameters: {
-                clientId: 'c9c45bfd-0ed3-4aa2-9971-40f88962b836',
-                organizationId: 'f_ecom_zzrf_001',
+                clientId: '199c814e-ce47-4af6-a699-b16e2c7b91e7',
+                organizationId: 'f_ecom_zzrf_009',
```

Comment:
Is this change intentional? Maybe I'm missing something but isn't it moving the default to no longer be our demo `zzrf_001` instance?

Reply:
Yeah, it was intentional to make it easier for the code reviewer to test the changes. I plan to remove it before the PR is merged.